gfx/UltimateCataclysm/pngs_large_64x64/monsters/mon_mech_combat/mon_mech_combat.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "id": "mon_mech_combat",
+    "fg": "mon_mech_combat",
+    "bg": ""
+  }
+]
```

Comment:
since the two-part idea didn't work I guess these JSONs are not required, no?

Reply:
I honestly don't know, I remember doing the lifter mech live and the first time I did it the normal version didn't have JSON but didn't appear. Might have messed something else up though. It's worth trying...